src/dolphin/manager.ts

```diff
@@ -46,6 +46,8 @@ export class DolphinManager extends EventEmitter {
       throw new Error("Netplay dolphin is already open!");
     }
 
+    void updateDolphinSettings();
```

Comment:
Why are we `void`ing these promises?

```suggestion
    await updateDolphinSettings();
```